src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
From what I can tell, windows command files are standardized and don't require the @ prefix because they can be handled by arbitrary executables, and are not specific to gcc/clang/ccache.  If this patch only gets applied to Windows, we shouldn't need to dissect the command at all.

I feel pretty strongly against copying in chunks of distutils to our code base.  I'd like to hear at least one other person say this is a reasonable approach for now.